test/codeu/chat/TestRunner.java

```diff
@@ -28,7 +28,8 @@ public static void main(String[] args) {
              codeu.chat.server.RawControllerTest.class,
              codeu.chat.util.TimeTest.class,
              codeu.chat.util.UuidTest.class,
-             codeu.chat.util.store.StoreTest.class
+             codeu.chat.util.store.StoreTest.class,
+             codeu.chat.util.TokenizerTest.class
```

Comment:
This file's permissions have been changed to mark it as executable, which is technically not right. I'd recommend changing this file back to its default permissions by running `chmod 0644 test/codeu/chat/TestRunner.java` if you're using Linux or Mac.